extension/src/background/messageListener/freighterApiMessageListener.ts

```diff
@@ -25,7 +21,6 @@ const WINDOW_DIMENSIONS = `width=${POPUP_WIDTH},height=667`;
 export const freighterApiMessageListener = (
   request: Request,
   sender: Sender,
-  sendResponse: (response: SendResponseInterface) => void,
```

Comment:
`webextension-polyfill` uses a promise based API rather than the callback based API we were using, so all these methods had to be mildly refactored